variables.tf

```diff
@@ -0,0 +1,50 @@
+variable "name" {}
```

Comment:
Use the same variables we have in all of our other modules.

Reply:
https://github.com/cloudposse/terraform-aws-key-pair/blob/master/variables.tf#L1-L32

They are all well documented